core/templates/dev/head/pages/state_editor/StateInteractionEditorDirective.js

```diff
@@ -188,16 +188,18 @@ oppia.directive('stateInteractionEditor', [
                 backdrop: true,
                 resolve: {},
                 controller: [
-                  '$scope', '$uibModalInstance', '$injector',
+                  '$scope', '$uibModalInstance', '$injector', '$window',
                   'StateSolutionService', 'StateInteractionIdService',
                   'StateCustomizationArgsService',
                   'InteractionDetailsCacheService', 'INTERACTION_SPECS',
+                  'ALLOWED_INTERACTION_CATEGORIES', 'ALLOWED_QUESTION_INTERACTION_CATEGORIES',
```

Comment:
Add this in sorted order. (Follow the pattern provided in #6004)

@anubhavsinha98  are we expecting this to be in order (#6125)?

Reply:
Hi @DubeySandeep 
Filed #6513 to address this. 
Thanks!

Reply:
Done.